Grammer on Home Page (#4)
Holben888/slinkity-home: src/index.md

Slinkity is a plugin that can extend any [11ty](https://11ty.dev/) site. Once installed, this:
- 🚀 **Unlocks component frameworks** (React, Vue, and Svelte) for writing page templates _and_ layout templates. So you can turn an existing `.html` or `.liquid` file into a `.jsx` file, and immediately start building routes on your site using React.
- 🔖 **Includes powerful shortcodes** to insert components into existing pages. Add a line like this to your markdown, HTML, Nunjucks, etc, and watch the magic happen: `{% raw %}{% react './path/to/component.jsx' %}{% endraw %}`

> - 💧 **Hydrates these component-driven pages** on the client. In other words, all your [dynamic state management](https://reactjs.org/docs/hooks-intro.html) will work in development and production with 0 extra setups.
> - 🔗 **(Optionally) Turns your site into a single page app.** This opens the door for animated page transitions, persistent states between pages, and more!

Reviewer: ```suggestion
- 💧 **Hydrates these component-driven pages** on the client. In other words, all your [dynamic state management](https://reactjs.org/docs/hooks-intro.html) will work in development and production with 0 extra setup.
- 🔗 **(Optionally) Turns your site into a single page app.** This opens the door for animated page transitions, persistent state between pages, and more!
```

Hm, I don't think the use of "plurals" here would be grammatically correct. I would prefer to keep this section as-is.

Let's see Slinkity in action ✨

{% include demo-embed.html %}

## Our philosophy

We built this tool to address an age-old question: **"what should I use to build my website?"**

This stumps newbies and professionals alike. In the static site space, we have 2 competing camps:

> 1. **The "simple" site generator** is driven by data and templating languages (markdown, HTML, liquid, etc). This approach avoids JavaScript bundles and component frameworks like React in favor of straight HTML and CSS. This camp is mostly dominated by [Jekyll](https://jekyllrb.com), [Hugo](https://gohugo.io), and our favorite, [11ty](https://11ty.dev).
> 2. **The "dynamic" site generator** is driven by component frameworks like React, Vue, and Svelte. These output simple HTML and CSS like the first camp, _along with_ a bundle of JS to enable dynamic user interactions. Think image carousels with arrow buttons, animated page transitions as you explore the site, and other assorted, code-driven whimsy. This camp includes [NextJS](https://nextjs.org), [GatsbyJS](https://www.gatsbyjs.com), [SvelteKit](https://kit.svelte.dev), and more.

Reviewer: ```suggestion
1. **The "simple" site generator,** driven by data and templating languages (markdown, HTML, liquid, etc). This approach avoids JavaScript bundles and component frameworks like React in favor of straight HTML and CSS. This camp is mostly dominated by [Jekyll](https://jekyllrb.com), [Hugo](https://gohugo.io), and our favorite, [11ty](https://11ty.dev).
2. **The "dynamic" site generator,** driven by component frameworks like React, Vue, and Svelte. These output simple HTML and CSS like the first camp, _along with_ a bundle of JS to enable dynamic user interactions. Think image carousels with arrow buttons, animated page transitions as you explore the site, and other assorted, code-driven whimsy. This camp includes [NextJS](https://nextjs.org), [GatsbyJS](https://www.gatsbyjs.com), [SvelteKit](https://kit.svelte.dev), and more.
```

These 2 sections were meant to be definitions for each type of SSG. An "is" wouldn't really work since we're coming from that colon from above, "In the static site space, we have 2 competing camps:"

Still, I agree a comma would break this up better!

> There's some clear value to both approaches. The first dramatically reduce build times and load times for the user by avoiding JavaScript processing. The second empowers your site to do _something more_ should you ever need it, at the cost of a large code bundle and steeper learning curve.

Reviewer: ```suggestion
There's some clear value to both approaches. The first dramatically reduces build times and load times for the user by avoiding JavaScript processing. The second empowers your site to do _something more_ should you ever need it, at the cost of a large code bundle and steeper learning curve.
```

"First" is singular, so changing to `reduce` wouldn't be proper admittedly. I'd prefer to keep this as-is.

But we're faced with a dilemma: **if we ever want to switch camps, we'll have to rewrite our site from scratch in a different tool.** This costs time, money, and user experience if anything goes wrong 😢

This is what **Slinkity** is built for. With our approach, you can:
1.  Start building your static site with 11ty. This means a lower learning curve, fast build times, and a strong developer community.

> 2.  When (or if) you need some dynamic user interactions, you're free to add React, Vue, or Svelte components to your existing site _with zero extra setups_. This could mean embedding components into existing templates using shortcodes (i.e. `{% raw %}{% react 'FancyComponent.js' %}{% endraw %}`), or **replacing** page and layout templates with components entirely (i.e. converting a liquid or HTML file into a JSX file).

Reviewer: ```suggestion
2.  When (or if) you need some dynamic user interactions, you're free to add React, Vue, or Svelte components to your existing site _with zero extra setup_. This could mean embedding components into existing templates using shortcodes (i.e. `{% raw %}{% react 'FancyComponent.js' %}{% endraw %}`), or **replacing** page and layout templates with components entirely (i.e. converting a liquid or HTML file into a JSX file).
```

Ditto on this one. Plural wouldn't get the point across right, since I'm essentially saying "there _is_ no extra setup."
